src/java.base/unix/classes/sun/nio/ch/UnixAsynchronousSocketChannelImpl.java

```diff
@@ -741,10 +741,12 @@ <V extends Number,A> Future<V> implWrite(boolean isGatheringWrite,
                 return result;
             }
         } catch (Throwable x) {
+            System.out.println("Caught " + x);
             if (x instanceof ClosedChannelException)
                 x = new AsynchronousCloseException();
             exc = x;
         } finally {
+            System.out.println("finally");
```

Comment:
As above

Reply:
This file was a leftover from a different issue I was looking at and was accidentally included here.

Reply:
Fixed in ff404d4b52c5ecf4f00743be1d4d4a266483460c.